frontend/src/components/Navigator/index.js

```diff
@@ -129,25 +135,31 @@ function Navigator({
           <div id="navigator-form">
             {page.review
             && page.render(
-              allComplete,
               formData,
               onFormSubmit,
               additionalData,
               onReview,
               approvingManager,
+              navigatorPages,
             )}
             {!page.review
             && (
               <Container skipTopPadding>
                 <NavigatorHeader
                   label={page.label}
                 />
+                {hasErrors
+                && (
+                  <Alert type="error" slim>
+                    Please complete all required fields before submitting this report.
+                  </Alert>
+                )}
                 <Form
                   onSubmit={handleSubmit(onContinue)}
                   className="smart-hub--form-large"
                 >
                   {page.render(additionalData, formData, reportId)}
-                  <Button type="submit" disabled={!isValid}>Continue</Button>
+                  <Button type="submit">Continue</Button>
                 </Form>
```

Comment:
It may be out of scope, but I think the submit button should show an alert if the specialist tries to submit a report that has already been submitted and is waiting for approval. Thoughts?

Reply:
That'll be taken care of in this [ticket](https://github.com/HHS/Head-Start-TTADP/issues/285)